<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeaderPage</class>
 <widget class="QWidget" name="HeaderPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="Line1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>670</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Line2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>78</y>
     <width>800</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16px;
background-color: rgb(115, 115, 115);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>TP1
网压</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Line1_2">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>0</y>
     <width>2</width>
     <height>78</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Line1_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>2</width>
     <height>78</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Line1_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>2</width>
     <height>78</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Line1_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>2</width>
     <height>78</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Line1_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>2</width>
     <height>78</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>51</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16px;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string>网流</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>121</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string>DC24V母线电压</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Voltage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>42</y>
     <width>51</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Current">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>42</y>
     <width>51</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_24VVoltage">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>42</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Date">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>91</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Time">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>42</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Positon">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>141</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="header_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CurrentFaultBtn">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>15</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/images/fault.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Level">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>42</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_speed">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>42</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Line1_7">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>2</width>
     <height>78</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="HelpitemBtn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>22</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(115, 115, 115);
border: 2px solid rgb(115, 115, 115);
font: 30pt &quot;黑体&quot;;
border-radius: 20px;
background-color: rgb(255, 242, 0);
border: 1px groove yellow;
</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLabel" name="Line1_8">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>0</y>
     <width>2</width>
     <height>78</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_limitspeed">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>42</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>0</y>
     <width>71</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16px;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string>限速值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Voltage_MC2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>42</y>
     <width>51</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Line1_9">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>0</y>
     <width>2</width>
     <height>78</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>51</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16px;
background-color: rgb(115, 115, 115);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>TP2
网压</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>header_background</zorder>
  <zorder>Line1</zorder>
  <zorder>Line2</zorder>
  <zorder>label</zorder>
  <zorder>Line1_2</zorder>
  <zorder>Line1_3</zorder>
  <zorder>Line1_4</zorder>
  <zorder>Line1_5</zorder>
  <zorder>Line1_6</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>headerlabel_Voltage</zorder>
  <zorder>headerlabel_Current</zorder>
  <zorder>headerlabel_24VVoltage</zorder>
  <zorder>headerlabel_Date</zorder>
  <zorder>headerlabel_Time</zorder>
  <zorder>headerlabel_Positon</zorder>
  <zorder>CurrentFaultBtn</zorder>
  <zorder>headerlabel_Level</zorder>
  <zorder>headerlabel_speed</zorder>
  <zorder>Line1_7</zorder>
  <zorder>HelpitemBtn</zorder>
  <zorder>Line1_8</zorder>
  <zorder>headerlabel_limitspeed</zorder>
  <zorder>label_4</zorder>
  <zorder>headerlabel_Voltage_MC2</zorder>
  <zorder>Line1_9</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
